Sources/JavaScriptKit/FundamentalObjects/JSClosure.swift

```diff
@@ -10,40 +11,9 @@ public protocol JSClosureProtocol: JSValueCompatible {
     func release()
 }
 
-/// `JSOneshotClosure` is a JavaScript function that can be called only once.
-public class JSOneshotClosure: JSObject, JSClosureProtocol {
```

Comment:
Could you measure runtime performance between `JSOneshotClosure` and `JSClosure` for deallocation overhead?
I think deallocation that depends on GC cycle will increase memory usage while waiting until event loop frame. 

(But I'm not sure how this overhead affects users experience 😅 )